### PR TITLE
New package: pahole-1.12

### LINKED ISSUE #10100: Package Request:  dwarves
Hi, is it possible to pack a new packages to void?

It is called  dwarves  in debian.

It is a set of utilities that help programmer to analyze the memory structure of the code.

Link to release and Packages. https://packages.debian.org/search?keywords=dwarves

### PR DESCRIPTION
**Pull request by abenson**
Closes #10100.

### CODE REVIEW

File: srcpkgs/pahole/template [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+short_desc="Various DWARF utils"
+maintainer="Andrew Benson <abenson+void@gmail.com>"
+license="GPL-2.0-only"
+homepage="http://git.kernel.org/?p=devel/pahole/pahole.git;a=summary"
+distfiles="https://git.kernel.org/pub/scm/devel/pahole/pahole.git/snapshot/pahole-${version}.tar.gz"
+checksum=2452464cbfdbaece6187d7860000fe8ef24b7271f88122cf540d3e0ab4a644d1
+
+pahole-devel_package() {
+	short_desc+=" - development files"
+
```
Comment (maxice8) [inline srcpkgs/pahole/template]: extra remove

**Comment by maxice8** [comment]
It is missing musl-obstack for *-musl arches

**Comment by abenson** [comment]
Any idea on how to get it to actually link to argp and obstack? Following examples from other templates is failing me.

**Comment by Johnnynator** [comment]
> Any idea on how to get it to actually link to argp and obstack? Following examples from other templates is failing me.

take a look at e.g. openlierox. cmake does not always obey the env LDFLAGS.

**Pull request merged**